### PR TITLE
Config changes

### PR DESCRIPTION
**Pull request by sohje**
## Short Description:
Detect config changes on the fly

## Fixes/Resolves/Closes (please use correct syntax):
- https://github.com/PokemonGoF/PokemonGo-Bot/issues/3588
- Some cleanup
- Need some tests on Win systems!
- Maybe we need some delay between checks?

### p.s. 
Im totally new to this project, so im not sure that bot logic isnt broken. But it works fine for me - OSx, bots running perfectly.

### CODE REVIEW

**Comment by doctordo0m** [comment]
IMHO this is far too much overhead for most of the people. This should definetly be an option in the config file, to be started or not (maybe even with a check frequency?)

**Comment by sohje** [comment]
@doctordo0m I dont think this option should be added to config.json file cause of self-reference. Maybe CLI option with delay param?

**Comment by k4n30** [comment]
@sohje merge conflicts needs resolving

File: pokecli.py [hunk 1]
```diff
@@ -32,18 +32,18 @@
 import os
 import ssl
 import sys
-reload(sys)
-sys.setdefaultencoding('utf8')
```
Comment (nivong) [inline pokecli.py]: why delete this?

**Comment by sohje** [comment]
Omg.. wrong click and F$#K

**Comment by danielsdian** [comment]
Would be nice if we could "Modularize" the config file in order to make it wasier to modify. Like, a section for GPS configs (GPS location, noise, altitude), another for pokemon management (capture, evolution and release), another for movement logic (Walk speeds, Spiral, cluster), stuff like that.

**Comment by k4n30** [comment]
@danielsdian different Feature request found here #3506 

This FR is for detecting config changes after bot start

**Comment by k4n30** [comment]
@sohje why the close?

**Pull request closed without merging**